packages/flame/lib/src/game/camera/camera_component.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:ui';
+import '../../../components.dart';
+import 'camera.dart';
+
+// TODO(st-pasha): extend from BaseComponent
+class CameraComponent {
```

Comment:
This should be named something differently now when it is not a component. CameraController or something maybe?

Reply:
As you remember, this class **is**, in fact, a `Component`, except that we temporarily orphaned it until we could figure out how it should interact with the `Game` and some other classes. Renaming it now will serve no purpose since we'd have to rename it back in the next PR.

Reply:
Since we don't have a solution for how to make it interact with the component tree when it is a component itself we will most likely release `rc14` before that has been fixed (we are planning to do this soon after the "Game as a component" PR is merged), and I don't think we want classes named `*Component` released if they can't be added to the component tree.

Reply:
Ok, fair enough. Renaming is easy anyway.